src/components/chart/chart.js

```diff
@@ -83,40 +91,6 @@ class HighchartsBaseChart {
         Highcharts.setOptions(chartOptions);
     };
 
-    // Utility function to merge two configs together
-    mergeConfigs = (baseConfig, newConfig) => {
```

Comment:
This has been moved to a utilities file now. We just need to be a bit careful when this is merged to ensure any other code that is using `this.mergeConfigs` gets updated.